smithy-aws-apigateway-openapi/src/main/java/software/amazon/smithy/aws/apigateway/openapi/CognitoUserPoolsConverter.java

```diff
@@ -28,32 +28,24 @@
 /**
  * An authentication scheme converter that adds Cognito User Pool based
  * authentication ({@code cognito_user_pools} to an OpenAPI model when the
- * {@code aws.cognito-user-pools} authentication scheme is found on
```

Comment:
Note: this became an auth trait, `cognitoUserPools`, and removes the need for a separate configuration trait.